static/app/views/alerts/rules/metric/details/anomalyDetectionFeedbackBanner.tsx

```diff
@@ -0,0 +1,92 @@
+import {Fragment, useCallback} from 'react';
+import styled from '@emotion/styled';
+
+import Alert from 'sentry/components/alert';
+import {useDismissable} from 'sentry/components/banner';
+import {Button} from 'sentry/components/button';
+import {feedbackClient} from 'sentry/components/featureFeedback/feedbackModal';
+import {t} from 'sentry/locale';
+import ConfigStore from 'sentry/stores/configStore';
+import type {Organization} from 'sentry/types/organization';
+import {trackAnalytics} from 'sentry/utils/analytics';
+import type {Incident} from 'sentry/views/alerts/types';
+
+interface AnomalyDetectionFeedbackProps {
+  id: string;
+  organization: Organization;
+  selectedIncident: Incident;
+}
+
+export default function AnomalyDetectionFeedbackBanner({
+  id,
+  organization,
+  selectedIncident,
+}: AnomalyDetectionFeedbackProps) {
+  const [isSubmitted, submit] = useDismissable(id);
+
+  const handleClick = useCallback(
+    (isAnomaly: boolean) => {
```

Comment:
Not blocking, but this variable name is unclear! Maybe `anomalyCorrectlyIdentified`?